pallets/did/src/tests.rs

```diff
@@ -135,18 +139,20 @@ fn check_successful_complete_creation() {
 	let auth_key = get_sr25519_authentication_key(true);
 	let alice_did = get_did_identifier_from_sr25519_key(auth_key.public());
 	let auth_did_key = did::DidVerificationKey::from(auth_key.public());
-	let enc_keys: BTreeSet<did::DidEncryptionKey> =
+	let enc_keys = BoundedBTreeSet::<did::DidEncryptionKey, <Test as Config>::MaxTotalKeyAgreementKeys>::try_from(
```

Comment:
Since we are generating DID creation details, the parameter should be `MaxNewKeyAgreementKeys `.
```suggestion
	let enc_keys = BoundedBTreeSet::<did::DidEncryptionKey, <Test as Config>::MaxNewKeyAgreementKeys >::try_from(
```